libraries/AdaptiveExpressions/FunctionUtils.cs

```diff
@@ -1090,13 +1184,13 @@ internal static (object, string) ConvertTimeZoneFormat(string timezone)
             return (convertedTimeZone, error);
         }
 
-        internal static (string, string) ReturnFormatTimeStampStr(DateTime datetime, string format)
+        internal static (string, string) ReturnFormatTimeStampStr(DateTime datetime, string format, CultureInfo locale)
         {
             string result = null;
             string error = null;
             try
             {
-                result = datetime.ToString(format, CultureInfo.InvariantCulture.DateTimeFormat);
+                result = datetime.ToString(format, locale);
             }
 #pragma warning disable CA1031 // Do not catch general exception types (capture any exception and return generic message)
             catch
```

Comment:
Do you want to disable this for the whole file?

Reply:
I don't think we should, each exception should have a justification, in many cases we don't seem to put enough thought on what exceptions we are trying to catch and we are actually handling them.

Reply:
I think we should disable this for the whole file. Is there any recommended way to do that?

Reply:
I'm fine with leaving per-instance.  I understand there is going to be a later clean-up looking at them all.

---
In reply to: [457435215](https://github.com/microsoft/botbuilder-dotnet/pull/4276#discussion_r457435215) [](ancestors = 457435215)